test/test-govet.sh

```diff
@@ -65,6 +65,19 @@ function consistent-imports() {
 	if grep $'\t"github.com/purpleidea/mgmt/engine/util"' "$1"; then	# import as engineUtil
 		return 1
 	fi
+
+	# import as golang.org/x/sys/unix
+	if grep -q -E $'\t\w*[[:space:]]*"syscall"' "$1"; then
```

Comment:
If we're doing it this way, rather than checking for the actual import, then we'd want `syscall.` (with a trailing explicit period, so maybe `\.` ?)

Reply:
Ah, I think I understand it now, updated!